src/helpers/RecordsProcessor.js

```diff
@@ -0,0 +1,43 @@
+import avro from 'avsc'
+import logger from './Logger'
+
+// Map records to decode Avro and return JSON as data to firehose.
+const processRecords = function (schema, records) {
+  let success = 0
+  let failure = 0
+  let type = avro.Type.forSchema(schema)
+  const output = records.map((record) => {
```

Comment:
You'll run into problems if `records` is not an array and you try to run `.map` on it. You should check to see if `records` is an array first.